resources/leiningen/new/brevity/src/brevity/cljc/routes.cljc

```diff
@@ -0,0 +1,19 @@
+(ns {{name}}.cljc.routes
+    (:require [bidi.bidi :as bidi]
+              [{{name}}.cljc.utils :as u]))
+
+(def page-routes
+  (u/router "/"
+            (u/page "" :index)
+            (u/context "blog"
+                       (u/page "/" :blog)
+                       (u/page ["/entry/" :id] :blog/entry))))
```

Comment:
What are your thoughts on namespaced keywords? I kinda hate how these keywords don't work super well in `let` in Clojure. For example,
```clojure
(let [{:keys [hello/world]} {:hello/world 4}] 
   hello/world)

; => CompilerException java.lang.RuntimeException: No such namespace: hello, compiling:(null:1:1)
```

Reply:
Also, kickass job with this DSL. Much clearer.

Reply:
Thanks!  I'm kind of on the fence - I can't think of too many cases where a map keyed by route keywords would be passed around, but if someone ran into such a case the compiler errors would be frustrating.  I think they would add a lot of value as a project grows, though - I like `:blog/themes/banner-image/upload` a lot better than `:blog-themes-banner-image-upload` or `:banner-image-upload`.

Reply:
It turns out `:blog/themes/banner-image/upload` isn't actually a valid keyword - so I'm gravitating towards just pure kebab keywords to minimize surprises for people.